vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher_server_list.h

```diff
@@ -71,19 +73,21 @@ class PublisherServerList {
       const ledger::Result result,
       const SharedServerPublisherPartial& list_publisher,
       const SharedPublisherBanner& list_banner,
-      ParsePublisherListCallback callback);
-
-  void SavePublishers(
-      const SharedServerPublisherPartial& list_publisher,
-      const SharedPublisherBanner& list_banner,
-      ParsePublisherListCallback callback);
+      ledger::ResultCallback callback);
 
   void SaveBanners(
+      const ledger::Result result,
       const SharedPublisherBanner& list_banner,
-      ParsePublisherListCallback callback);
+      ledger::ResultCallback callback);
+
+  void BannerSaved(
+      const ledger::Result result,
+      ledger::ResultCallback callback);
 
   bat_ledger::LedgerImpl* ledger_;  // NOT OWNED
   uint32_t server_list_timer_id_;
+  bool in_progress_;
+  uint32_t current_page_;
```

Comment:
Should we initialize these on construction just to be safe?

Reply:
I think there is no real point as when you call to download you always need to reset it as you can have multiple downloads while the same instance is running. I can easily add it, but I don't think it will improve any

Reply:
I'm good either way, but it could be momentarily confusing if you're in the debugger for example and they fill randomly.

Reply:
that makes sense let me add them